src/protocols/hpr_protocol_router.erl

```diff
@@ -1,12 +1,175 @@
 -module(hpr_protocol_router).
 
+-include("grpc/autogen/server/packet_router_pb.hrl").
+
+% TODO: should be in a common include file
+-define(JOIN_REQUEST, 2#000).
```

Comment:
```suggestion
-include("semtech_udp.hrl").
```

Reply:
done

Reply:
`JOIN_REQUEST` isn't in `semtech_udp.hrl`. Is the suggestion to move it there?

Reply:
mmm, brain cross again. These are LoRa values, not Semtech UDP values. You're correct. I'm unsure where they should go right now.